specification/storage/resource-manager/Microsoft.Storage/stable/2021-09-01/storage.json

```diff
@@ -2620,6 +2623,13 @@
           "format": "date-time",
           "description": "Timestamp of last rotation of the Key Vault Key.",
           "x-ms-client-name": "LastKeyRotationTimestamp"
+        },
+        "currentVersionedKeyExpirationTimestamp": {
```

Comment:
The name is so long. .net SDK generation looks not correct with the name.
Could we make the name short?

Reply:
It is an existing property, which we return today by REST service and we are adding this to the Swagger to attain parity between SDK and REST. Check this
https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=613714BB-D7F0-4132-A458-45127F4D6B46

So we cannot change the name now